---
float32 torsion_modulus
